common/types/data_connections.ts

```diff
@@ -13,6 +13,15 @@ export interface PermissionsConfigurationProps {
   hasSecurityAccess: boolean;
 }
 
+export interface AssociatedObject {
+  id: string;
+  name: string;
+  database: string;
+  type: string;
+  createdByIntegration: string;
+  accelerations: string;
```

Comment:
would any of these fields be a smaller type than `string`, or all of them can be arbitrary strings?